sim/target-agnostic.mk

```diff
@@ -183,35 +190,67 @@ $(f1): $(header) $(DRIVER_CC) $(DRIVER_H) $(midas_cc) $(midas_h)
 	GEN_DIR=$(OUTPUT_DIR)/build OUT_DIR=$(OUTPUT_DIR) DRIVER="$(DRIVER_CC)" \
 	TOP_DIR=$(chipyard_dir)
 
+$(vitis): export CXXFLAGS := $(CXXFLAGS) $(common_cxx_flags) $(DRIVER_CXXOPTS)
+# Statically link libfesvr to make it easier to distribute drivers to f1 instances
+$(vitis): export LDFLAGS := $(LDFLAGS) $(common_ld_flags)
+
+# Compile Driver
+$(vitis): $(header) $(DRIVER_CC) $(DRIVER_H) $(midas_cc) $(midas_h)
+	mkdir -p $(OUTPUT_DIR)/build
+	cp $(header) $(OUTPUT_DIR)/build/
+	# The manager expects to find the default conf in output/ by this name
+	cp -f $(GENERATED_DIR)/$(CONF_NAME) $(OUTPUT_DIR)/runtime.conf
+	$(MAKE) -C $(simif_dir) vitis PLATFORM=vitis DRIVER_NAME=$(DESIGN) GEN_FILE_BASENAME=$(BASE_FILE_NAME) \
+	GEN_DIR=$(OUTPUT_DIR)/build OUT_DIR=$(OUTPUT_DIR) DRIVER="$(DRIVER_CC)" \
+	TOP_DIR=$(chipyard_dir)
+
 #############################
 # FPGA Build Initialization #
 #############################
-board_dir 	   := $(fpga_dir)/hdk/cl/developer_designs
+ifeq ($(PLATFORM), vitis)
+board_dir 	   := $(platforms_dir)/vitis
+else
+board_dir 	   := $(platforms_dir)/f1/aws-fpga/hdk/cl/developer_designs
+endif
+
 fpga_work_dir  := $(board_dir)/cl_$(name_tuple)
 fpga_build_dir := $(fpga_work_dir)/build
 verif_dir      := $(fpga_work_dir)/verif
 repo_state     := $(fpga_work_dir)/design/repo_state
+fpga_driver_dir:= $(fpga_work_dir)/driver
 
 # Enumerates the subset of generated files that must be copied over for FPGA compilation
 fpga_delivery_files = $(addprefix $(fpga_work_dir)/design/$(BASE_FILE_NAME), \
 	.sv .defines.vh .env.tcl \
-	.synthesis.xdc .implementation.xdc \
-	.ila_insert_inst.v .ila_insert_ports.v .ila_insert_wires.v .ila_insert_vivado.tcl)
+	.synthesis.xdc .implementation.xdc)
```

Comment:
IIRC we were just going to blindly copy all files needed to generate Vitis/F1 bitstreams (if the file wasn't generated, then silently continue).